diff --git a/arch/arm/dts/rk3568-h96max-v56-u-boot.dtsi b/arch/arm/dts/rk3568-h96max-v56-u-boot.dtsi
new file mode 100644
index 00000000000..3cc637d2b7c
--- /dev/null
+++ b/arch/arm/dts/rk3568-h96max-v56-u-boot.dtsi
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+
+#include "rk356x-u-boot.dtsi"
+
+/ {
+	chosen {
+		stdout-path = &uart2;
+	};
+};
+
+&emmc_bus8 {
+	bootph-all;
+};
+
+&emmc_clk {
+	bootph-all;
+};
+
+&emmc_cmd {
+	bootph-all;
+};
+
+&emmc_datastrobe {
+	bootph-all;
+};
+
+&fspi_pins {
+	bootph-all;
+};
+
+&pcfg_pull_none {
+	bootph-all;
+};
+
+&saradc {
+	status = "okay";
+};
+
+&sdmmc0_bus4 {
+	bootph-all;
+};
+
+&sdmmc0_clk {
+	bootph-all;
+};
+
+&sdmmc0_cmd {
+	bootph-all;
+};
+
+&sdmmc0_det {
+	bootph-all;
+};
+
+&sfc {
+	bootph-pre-ram;
+	u-boot,spl-sfc-no-dma;
+
+	flash@0 {
+		bootph-pre-ram;
+		spi-rx-bus-width = <4>;
+		spi-tx-bus-width = <1>;
+	};
+};
+
+&uart2 {
+	clock-frequency = <24000000>;
+	bootph-all;
+	status = "okay";
+};
+
+&uart2m0_xfer {
+	bootph-all;
+};
diff --git a/arch/arm/dts/rk3568-h96max-v56.dts b/arch/arm/dts/rk3568-h96max-v56.dts
new file mode 100644
index 00000000000..7d2ca16f5f7
--- /dev/null
+++ b/arch/arm/dts/rk3568-h96max-v56.dts
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Minimal U-Boot DT for H96 TV Box 3566 with eMMC, SD-card,
+ * SPI flash and USB OTG enabled.
+ */
+
+/dts-v1/;
+
+#include "rk3568.dtsi"
+
+/ {
+	model = "h96 TVbox 3566";
+	compatible = "h96-TVbox,rk3566", "rockchip,rk3566";
+
+	aliases {
+		mmc0 = &sdhci;
+		mmc1 = &sdmmc0;
+	};
+};
+
+&sdhci {
+	no-sd;
+	no-sdio;
+	non-removable;
+};
+
+&sdmmc0 {
+	no-mmc;
+	no-sdio;
+};
-- 
2.49.0

